.gitlab-ci.yml

```diff
@@ -35,10 +31,10 @@ stages:
         --timeout 120s
         --create-namespace
         --namespace $KUBE_NAMESPACE
-        --set image.backend.repository="${CONTAINER_REGISTRY}/${BACKEND_CONTAINER_REPO}"
-        --set image.backend.tag="${CI_COMMIT_SHORT_SHA}"
-        --set image.frontend.repository="${CONTAINER_REGISTRY}/${FRONTEND_CONTAINER_REPO}"
-        --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}"
+        --set image.backend.repository="${BACKEND_CONTAINER_REPO}"
+        --set image.backend.tag="${CI_COMMIT_SHORT_SHA}-beta"
+        --set image.frontend.repository="${FRONTEND_CONTAINER_REPO}"
+        --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}-beta"
```

Comment:
why beta?

Reply:
it's for the staging environment and the code might be broken. I labeled them with `-beta` so everyone would know that it's neither the `stable` nor the `latest` version of the docker image.

Later another issue could be opened in the CI/CD repository to tag the docker images based on the tags provided by the developers if needed.